lib/jekyll/webmention_io.rb

```diff
@@ -300,19 +301,22 @@ def self.log(type, message)
     end
 
     # Utility Method
-    # Writes given +data+ as YAML string into +file+ path.
+    # Caches given +data+ to memory and then proceeds to write +data+
+    # as YAML string into +file+ path.
     #
     # Returns nothing.
     def self.dump_yaml(file, data = {})
+      @webmention_data_cache[file] = data
       File.open(file, "wb") { |f| f.puts YAML.dump(data) }
```

Comment:
Is there a way to async this operation so it doesn’t block?

Reply:
Which of the two operations is the blocker..?
Either ways, `:dump_yaml` is called just 3-4 times in a build, so it shouldn't cause any noticeable bottlenecks..

Reply:
I was figuring the disk write might do some blocking. But maybe it doesn’t.